Doc/library/stdtypes.rst

```diff
@@ -1622,7 +1622,15 @@ expression support in the :mod:`re` module).
 .. method:: str.capitalize()
 
    Return a copy of the string with its first character capitalized and the
-   rest lowercased.
+   rest lowercased::
+
+      >>> 'PYTHON IS AMAZING'.capitalize()
+      'Python is amazing'
+      >>> 'ǋemačka Starts With a non-English Digraph'.capitalize()
```

Comment:
Starting with a lowercase digraph would make a better example:
```suggestion
      >>> 'ǌemačka Starts With a non-English Digraph'.capitalize()
```

Reply:
Thank you!